_posts/convars/2021-12-09-snd_volume_weapon_overall.md

```diff
@@ -0,0 +1,16 @@
+---
+title: snd_volume_weapon_overall
+category: var
+tags:
+  - sound
+  - volume
+  - weapon
+minimum_value: 0
+maximum_value: 1
+default_value: 1
+---
+
+{:.notice--info}
+Unlike [snd_volume_weapon_player]({% post_url 2021-12-09-snd_volume_weapon_player %}), this cvar is for _any_ weapon sounds
```

Comment:
Best to not refer to this with the date and instead use a relative path, like
```
[`snd_volume_weapon_player`](/var/snd_volume_weapon_player)
```
Also we've been putting cvar names in code blocks with backticks `

Reply:
Resolved in [`bf10b7f` (#229)](https://github.com/momentum-mod/docs/pull/229/commits/bf10b7f09a7a25318c2573a220ffe77a5aa73709)